.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature request
+about: Suggest an idea to discuss
+title: "[Feature Request]"
+labels: feature-request
+assignees: ''
+
+---
+
+_A new feature is some behavior that is not existing already._
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
Can we take out the phrase "clear and concise" in both places?

Reply:
removed from all three.